src/test/js/shared/random_data.js

```diff
@@ -44,7 +44,7 @@ function shuffleArray(array) {
     return array;
 }
 
-const testBasePrefix = "SIDMTESTWP_" + randomAlphabeticString();
+const testBasePrefix = "iwp_" ;
```

Comment:
why did we decide to change our usual prefix?

Reply:
It was mentioned in the jira